lib/earmark/transform.ex

```diff
@@ -86,10 +86,25 @@ defmodule Earmark.Transform do
   defp escape("", _opions) do
     []
   end
-  defp escape(element, options) do
-    element
-      |> smartypants(options)
-      |> Earmark.Helpers.escape(true)
+
+  @dbl1_rgx ~r{(^|[-–—/\(\[\{"”“\s])'}
```

Comment:
I am always nervous about `̂ and `$` in regexen so I tend to use `\A`  and `\z` so that adding the multiline flag to the regex cannot break it